client/src/pages/About.tsx

```diff
@@ -1,8 +1,231 @@
 import React from 'react';
-import { PageWrapper } from '../components';
+import styled, { css } from 'styled-components';
+import {
+  H1,
+  H2,
+  P,
+  PageWrapper,
+  SubTitle,
+} from '../components';
+import { COLORS } from '../constants';
+import Logo from '../imgs/logo.png';
+import erinPic from '../imgs/erinPic.png';
+import andreiPic from '../imgs/andreiPic.png';
+import jodiPic from '../imgs/jodiPic.png';
+
+const Box_Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  margin-top: 1900px;
```

Comment:
There's a bug with the PageWrapper- You shouldn't need this `margin-top` here. It's fine for now, but we need to look into that in the future.

Reply:
Oops I lied - You shouldn't have `height: 85vh;` in your `FlexBoxWrapper`. You shouldn't be basing your height based on vh for this page because it's meant to be scrollable

Reply:
You also shouldn't have `justify-content: center` because that is centering all of your content vertically